src/main/java/seedu/address/logic/parser/Parser.java

```diff
@@ -273,6 +273,7 @@ private RecurringType checkForRecurringTask(String args) throws IllegalArgumentE
     private Command prepareAddNonFloatingByDate(Matcher matcher) throws IllegalValueException {
         String endInput = matcher.group("deadline");
         RecurringType recurringType = checkForRecurringTask(endInput);
+        if(recurringType == RecurringType.IGNORED) recurringType = RecurringType.NONE;
```

Comment:
Hmm I think more can be done regarding the coding standard? 
Instead of if(recurringType == RecurringType.IGNORED) recurringType = RecurringType.NONE;
Shouldn't it be if(recurringType == RecurringType.IGNORED) {
                             recurringType = RecurringType.NONE;
                         }